src-java/kilda-model/src/main/java/org/openkilda/model/FlowStats.java

```diff
@@ -156,17 +178,38 @@ public String getFlowId() {
     public interface FlowStatsCloner {
         FlowStatsCloner INSTANCE = Mappers.getMapper(FlowStatsCloner.class);
 
-        void copy(FlowStatsData source, @MappingTarget FlowStatsData target);
-
         @Mapping(target = "flowObj", ignore = true)
+        @Mapping(target = "haSubFlowObj", ignore = true)
         void copyWithoutFlow(FlowStatsData source, @MappingTarget FlowStatsData target);
 
+        /**
+         * Performs copy of entity data.
+         */
+        default void copy(FlowStatsData source, FlowStatsData target) {
+            if (source == null) {
+                return;
+            }
+
+            if (source.getFlowObj() != null) {
+                target.setFlowObj(source.getFlowObj());
+            } else {
+                target.setHaSubFlowObj(source.getHaSubFlowObj());
+            }
+            target.setForwardLatency(source.getForwardLatency());
+            target.setReverseLatency(source.getReverseLatency());
+            target.setTimeModify(source.getTimeModify());
```

Comment:
I think here you should reuse copyWithoutFlow, because when someone will add a new field onto stats object he can forget to add field here. 
Also, if you will copy object like `target.setFlowObj(source.getFlowObj());` it will copy just a link on the old Flow object.

Here is an example how we can do it https://github.com/telstra/open-kilda/blob/df82665a9655cb3f339800fe3a5a435102ea9eed/src-java/kilda-model/src/main/java/org/openkilda/model/Port.java#L143

Reply:
Updated. Thanks!